gradle-plugin/src/main/kotlin/org/mozilla/android/MozillaPlugin.kt

```diff
@@ -6,9 +6,34 @@ package org.mozilla.android
 
 import org.gradle.api.Plugin
 import org.gradle.api.Project
+import org.gradle.api.artifacts.repositories.MavenArtifactRepository
+import java.net.URI
 
 class MozillaPlugin : Plugin<Project> {
 
     override fun apply(project: Project) {
+        with(project) {
```

Comment:
So this only does it for the one project.  That's generally the right thing to do for a general plugin.  But we're trying to help people so we might want this to be `project.rootProject.allprojects { ... }`.

@mcomella what's your take?

Reply:
I expect users of this plugin to do something like `allprojects { apply 'org.mozilla.android' }` - and that's how I tested it locally; adding this plugin to a-c is all that seems to be necessary. I didn't have to add the plugin to consumers of a-c.

Reply:
Good observation. I think we should only apply the transformation on the modules the plugin is applied to (it seems the most correct).

The downside is that consumers might not be aware of how plugins typically work and run into trouble. I think we could add Grisha's suggestion (`allprojects { apply`) as the suggested application method to the readme to make sure we make this as easy as possible.